src/etc/inc/auth.inc

```diff
@@ -33,6 +33,7 @@
 
 require_once("interfaces.inc");
 require_once("util.inc");
+require_once("config.inc");
```

Comment:
This looks wrong. The other too, historically, are wrong as well. Which script misses what?

Reply:
Looks like sync_user.php was including auth.inc and config.inc in the wrong order - auth.inc calls functions when loaded which depend on the config already being loaded.  I've updated the PR now, maybe that's less wrong?

Reply:
Yes, nice catch ❤️